### PR TITLE
Plus circle click

### PR DESCRIPTION
**Pull request by huwshimi**
Show the mid-point when the canvas plus circle is clicked.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- load `http://0.0.0.0:6543/:flags:/react/`
- click the orange (+) in the centre of the canvas
- the mid-point should appear
- load `http://0.0.0.0:6543/:flags:/react/` again
- in your browser console run `app.get('charmstore').getEntity('precise/python-django', function(entity) { app.deployService(entity[0]); });`
- a new orange (+) should appear somewhere on the canvas
- click it
- the mid-point should appear

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3106/
Test PASSed.

**Comment by anthonydillon** [comment]
Code and QA are good :+1:

File: jujugui/static/gui/src/app/views/environment.js [hunk 1]
```diff
@@ -211,6 +211,16 @@ YUI.add('juju-view-environment', function(Y) {
           helpText.show();
           helpText.removeClass('shrink');
           includedPlus.classed('show', false);
+          helpText.one('.plus-service').on('click', function() {
```
Comment (hatched) [inline jujugui/static/gui/src/app/views/environment.js]: This is going to attach this click event every time there is a service update in the db so this needs to move elsewhere.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3123/
Test PASSed.

**Comment by hatched** [comment]
:+1: QA OK!

**Comment by huwshimi** [comment]
Thanks @hatched and @anthonydillon :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**